api/players/overview/types.ts

```diff
@@ -89,8 +89,8 @@ export interface ProfileOverview {
 			name: string
 			competition: string
 		}
-		altPosition: string
-		position: string
+		altPosition: number
+		position: number
```

Comment:
we still need the name for previous stuff😅
```suggestion
		altPosition: {
		name: string
		gridPos: number
		}
		position: {
		name: string
		gridPos: number
		}
```